BUILD.md

```diff
@@ -326,17 +362,3 @@ ls -l /code/onnxruntime/build/Linux/MinSizeRel/dist/*.whl
 
 ### Using other compilers
 (TODO)
-
```

Comment:
is this supposed to be removed?

Reply:
Hey George, 
We didnt make this change

Reply:
oh sorry didnt see this properly i thought you were referring to the TODO

Reply:
yes, i was referring to the Android Builds/Cross compiling on Linux sections. Did those get removed accidentally?

Reply:
Yes. Looks like this content was accidentally remove. It is restored back now.